docs/azure/ai/get-started-app-chat-template.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Get started with the enterprise chat app template for .NET
+description: Get started with .NET and search across your own data using an Azure OpenAI chat app. Easily deploy with Azure Developer CLI. This article uses the Azure AI Reference Template sample.
+ms.date: 10/30/2023
+ms.topic: get-started
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact with app infused with my own business data so that learn from the sample code.
+---
+
+# Get started with the enterprise chat app template for .NET
+
+Deploy and use a chat app to get answers about employee benefits at a fictitious company with .NET. The employee benefits chat app is seeded with PDF files including the employee handbook, a benefits document and a list of company roles and expectations.
+
+> [!div class="nextstepaction"]
+> [Begin now](#open-development-environment)
+
+By following the instructions in this article, you will:
+
+- Deploy a chat app to Azure.
+- Get answers about employee benefits.
+- Change settings to change behavior of responses.
+
+Once you complete this procedure,you can start modifying the new project with your custom code.
+
+This article is part of a collection of articles that show you how to build a chat app using Azure Cognitive Search and OpenAI.
+
+Other articles in the collection include:
+
+* [Python](/azure/developer/python/get-started-app-chat-template)
+* [JavaScript](/azure/developer/javascript/get-started-app-chat-template)
+* [Java](/azure/developer/java/get-started-app-chat-template)
+
+## Architectural overview
+
+A simple architecture of the chat app is shown in the following diagram:
+
+:::image type="content" source="../media/get-started-app-chat-template/simple-architecture-diagram.png" alt-text="Diagram showing architecture from client to backend app.":::
+
+Key components of the architecture include:
+
+* A web application to host the interactive chat experience.
+* An Azure Cognitive Search resource to get answers from your own data.
+* An Azure OpenAI Service to provide:
+  * Keywords to enhance the search over your own data.
+  * Answers from the OpenAI model.
+  * Embeddings from the ada model
+
+## Cost
+
+Most resources in this architecture use a basic or consumption pricing tier. Consumption pricing is based on usage, which means you only pay for what you use. To complete this article, there will be a charge but it will be minimal. When you are done with the article, you can delete the resources to stop incurring charges.
```

Comment:
Is this still true, or is there a free version?

Reply:
There is no free version to my knowledge